Assets/Scripts/Game/TalkManager.cs

```diff
@@ -1922,16 +1946,27 @@ public void RestoreConversationData(SaveDataConversation data)
 
                         string key = item.Symbol.Name;
 
-                        if (questInfo.resourceInfo.ContainsKey(key)) // if orphaned list of quest resources contains a matching entry
+                        if (questInfo.resourceInfo.ContainsKey(key)) // if list of quest resources contains a matching entry
                         {
                             questInfo.resourceInfo[key].questResource = item; // update (relink) it
                         }
                     }
 
                 }
             }
-
             listRumorMill = data.listRumorMill;
+
+            // search for orphaned entries in rumor mill
+            for (int i = listRumorMill.Count - 1; i >= 0; i--)
+            {
+                ulong questID = listRumorMill[i].questID;
+                if (GameManager.Instance.QuestMachine.GetQuest(questID) == null)
+                {
+                    Debug.Log(String.Format("save data contains orphaned rumors for quest with id {0}. Removing these rumors...", questID));
+                    listRumorMill.Remove(listRumorMill[i]);
+                }
+            }
+
             if (listRumorMill == null)
```

Comment:
Update: Oh, you have this null check already - sorry mate. Just need to move this up above the for loop then. :)

Reply:
thanks for improving this ;)